[2022-05-22 03:01:08,909] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Teste_ETL_Raízen.extract_tables_file manual__2022-05-22T02:35:48.168632+00:00 [queued]>
[2022-05-22 03:01:08,917] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: Teste_ETL_Raízen.extract_tables_file manual__2022-05-22T02:35:48.168632+00:00 [queued]>
[2022-05-22 03:01:08,917] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-05-22 03:01:08,917] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-05-22 03:01:08,918] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-05-22 03:01:08,929] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): extract_tables_file> on 2022-05-22 02:35:48.168632+00:00
[2022-05-22 03:01:08,934] {standard_task_runner.py:52} INFO - Started process 792 to run task
[2022-05-22 03:01:08,936] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'Teste_ETL_Raízen', 'extract_tables_file', 'manual__2022-05-22T02:35:48.168632+00:00', '--job-id', '46', '--raw', '--subdir', 'DAGS_FOLDER/ETL_Raizen.py', '--cfg-path', '/tmp/tmpm2o4shv1', '--error-file', '/tmp/tmp93vdi4gy']
[2022-05-22 03:01:08,937] {standard_task_runner.py:80} INFO - Job 46: Subtask extract_tables_file
[2022-05-22 03:01:08,976] {task_command.py:369} INFO - Running <TaskInstance: Teste_ETL_Raízen.extract_tables_file manual__2022-05-22T02:35:48.168632+00:00 [running]> on host de43a65d8e31
[2022-05-22 03:01:09,035] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=Teste_ETL_Raízen
AIRFLOW_CTX_TASK_ID=extract_tables_file
AIRFLOW_CTX_EXECUTION_DATE=2022-05-22T02:35:48.168632+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2022-05-22T02:35:48.168632+00:00
[2022-05-22 03:02:48,457] {python.py:173} INFO - Done. Returned value was: None
[2022-05-22 03:02:48,468] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=Teste_ETL_Raízen, task_id=extract_tables_file, execution_date=20220522T023548, start_date=20220522T030108, end_date=20220522T030248
[2022-05-22 03:02:48,516] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-05-22 03:02:48,540] {local_task_job.py:273} INFO - 1 downstream tasks scheduled from follow-on schedule check
